imap_processing/tests/lo/test_bit_decompression.py

```diff
@@ -11,9 +11,9 @@
 @pytest.mark.parametrize(
     ("decompression", "compressed_value", "expected_decompressed_int"),
     [
-        (Decompress.DECOMPRESS8TO16, [209, 250], [9537, 53016]),
-        (Decompress.DECOMPRESS8TO12, [205], [1229]),
-        (Decompress.DECOMPRESS12TO16, [4068], [62908]),
+        (Decompress.DECOMPRESS8TO16, [209, 250], [13426, 54336]),
```

Comment:
Quick question: why did the values in the tests change?

Reply:
These tables are using a different compression so the decompressed values are different